doc/using/InvokingGHDL.rst

```diff
@@ -500,6 +500,19 @@ Options
   Currently, VITAL checks are only partially implemented.
   See section :ref:`VHDL_restrictions_for_VITAL` for more details.
 
+.. option:: --check-synthesis
```

Comment:
I am wondering if the name of the flag is the right one.  As this is a flag for warnigs, I would propose something like  `-Wsynthesis` (or `-Wsynth-sensitivity-list` if not too long).

Then you would be able to directly use the warning mechanisms: Use `Is_Warning_Enabled` to test if this option is present.

Reply:
Hi, this has been actually done, "Warnid_Synth_Sensitivity_Lists" has been added and by default it is enabled.

The idea is that `--check-synthesis` will enable all Synthesis checks and "Warnid_Synth_Sensitivity_Lists" will
be a sub-set of these checks. Currently, since Sensitivity list check is the only one, these two options are equal.
Later, I would like to add others.

Reply:
I am fine with multiple flags.  But still I think all command line flags for warnings must start with `-W` to easily identify them as warnings.  So what about `-Wsynthesis-sensitivity` for this issue and `-Wsynthesis` to enable all warnings about synthesis ?

Reply:
Hi, thanks. It sounds reasonable. Plus, if the nested options will have the same prefix as the "master" option, it will be clear that these belong together.